index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+   <div class="content">
+    <section class="top_section">
+<div class="image_container">
+    <img src="aaa.jpg" alt="tribute">
+</div>
+<div>
+    <h1> SPIDER MAN </h1>
+    <h4>(MARVAL STUDIO)</h4>
+</div>
+
+    </section>
+    <section class="about_section">
+   <h2>MILES MORALES</h2>
+    <p>
+        <b>Marvel's Spider-Man: Miles Morales is a 2020</b> action-adventure game developed by Insomniac Games and published by Sony Interactive Entertainment. Based on the Marvel Comics character Miles Morales, it is inspired by both the character's decade-long comic book mythology and appearances in other media. The game is a spin-off and continuation of Marvel's Spider-Man (2018), and the second installment in Insomniac's Spider-Man series. The narrative focuses on Miles' struggle to balance the duties of his civilian persona and his role as the second Spider-Man when his new home, Harlem, is threatened by a war between the Roxxon Energy Corporation and a high-tech criminal army called the Underground, led by the mysterious Tinkerer.
+   </p>
+    </section>
+    <section class="biography_section">
+        <h3>biographoes</h3>
+        <ul>
+            <li>Gifted with web-shooters and guided by Peter Parker, Miles Morales uses his abilities to be a different kind of Spider-Man.
+            </li>
+        </ul>
+        <ul>
```

Comment:
Consider consolidating the biography points into a single ul list for better organization and readability